cvat/apps/iam/templates/account/email/email_confirmation_message.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+{% load account %}{% user_display user as user_display %}{% load i18n %}{% autoescape off %}
+{% load static %}
+
+<html>
+<head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title>Email Confirmation</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style type="text/css">
```

Comment:
please make sure the template is correctly rendered in the popular email clients like gmail, apple mail, etc